internal/helpers/testsacc/uuid_test.go

```diff
@@ -0,0 +1,35 @@
+// SPDX-FileCopyrightText: Copyright (c) 2025 Orange
+// SPDX-License-Identifier: Mozilla Public License 2.0
+//
+// This software is distributed under the MPL-2.0 license.
+// the text of which is available at https://www.mozilla.org/en-US/MPL/2.0/
+// or see the "LICENSE" file for more details.
+
+package testsacc
+
+import (
+	"testing"
+)
+
+func TestIsUUIDv4(t *testing.T) {
+	tests := []struct {
+		name    string
+		input   string
+		wantErr bool
+	}{
+		{"valid uuid v4", "b3b6a6e2-7e6b-4c2a-8e2b-2b7e6b4c2a8e", false},
```

Comment:
Use google uuid package to create real valid uuidv4

```suggestion
		{"valid uuid v4", uuid.New(), false},
```